plugins/module_utils/identity/keycloak/keycloak.py

```diff
@@ -3011,3 +3014,11 @@ def get_authz_policy_by_name(self, name, client_id, realm):
                                                  validate_certs=self.validate_certs).read()))
         except Exception:
             return False
+
+    def fail_open_url(self, e, msg, **kwargs):
+        try:
+            if isinstance(e, HTTPError):
+                msg += ": " + e.read().decode()
```

Comment:
Instead of calling `decode()`, better use `to_native()`:
```suggestion
                msg += ": " + to_native(e.read())
```
Also formatting this is potentially a bit more efficient:
```suggestion
                msg = "%s: %s" % (msg, to_native(e.read()))
```